syncd/syncd.cpp

```diff
@@ -2535,41 +2535,83 @@ sai_status_t processEvent(
     return status;
 }
 
-void processFlexCounterEvent(
-        _In_ swss::ConsumerStateTable &consumer)
+void processFlexCounterGroupEvent(
+        _In_ swss::ConsumerTable &consumer)
 {
     SWSS_LOG_ENTER();
 
     std::lock_guard<std::mutex> lock(g_mutex);
-
     swss::KeyOpFieldsValuesTuple kco;
     consumer.pop(kco);
 
     const auto &key = kfvKey(kco);
     const auto &op = kfvOp(kco);
+    const auto values = kfvFieldsValues(kco);
 
-    std::size_t delimiter = key.find_last_of(":");
-
-    if (delimiter == std::string::npos)
+    if (op == DEL_COMMAND)
     {
-        SWSS_LOG_ERROR("Failed to parse the key %s", key.c_str());
+        FlexCounter::removeCounterPlugin(key);
         return;
     }
 
-    const auto intervalStr = key.substr(delimiter + 1);
-    const auto vidStr = key.substr(0, delimiter);
-    int pollInterval;
-
-    try
+    for (const auto& valuePair : values)
     {
-        pollInterval = std::stoi(intervalStr);
+        const auto field = fvField(valuePair);
+        const auto value = fvValue(valuePair);
+
+        if (op == SET_COMMAND)
+        {
+            if (field == POLL_INTERVAL_FIELD)
+            {
+                FlexCounter::setPollInterval(stoi(value), key);
+            }
+            else if (field == QUEUE_PLUGIN_FIELD)
+            {
+                auto shaStrings = swss::tokenize(value, ',');
+                for (const auto &sha : shaStrings)
+                {
+                    FlexCounter::addQueueCounterPlugin(sha, key);
+                }
+            }
+            else if (field == PORT_PLUGIN_FIELD)
+            {
+                auto shaStrings = swss::tokenize(value, ',');
+                for (const auto &sha : shaStrings)
+                {
+                    FlexCounter::addPortCounterPlugin(sha, key);
+                }
+            }
+            else
+            {
+                SWSS_LOG_ERROR("Object type not supported");
```

Comment:
>SWSS_LOG_ERROR("Object type not supported"); [](start = 16, length = 44)

should be field not supported, also please print out the actual field for debugging purpose.